khal/configwizard.py

```diff
@@ -153,21 +160,38 @@ def get_vdirs_from_vdirsyncer_config():
             path += '*'
             vdirs.append((storage['instance_name'], path, 'discover'))
     if vdirs == []:
-        print("No usable collections were found")
+        print("No calendars found from vdirsyncer.")
         return None
     else:
-        print("The following collections were found:")
-        for name, path, _ in vdirs:
-            print(f'  {name}: {path}')
         return vdirs
 
 
+def find_vdir():
+    """Use one or more existing vdirs on the system.
+
+    Tries to get data from vdirsyncer if it's installed and configured, and
+    asks user to confirm it. If not, prompt the user for the path to a single
+    vdir.
+    """
+    print("The following collections were found:")
+    synced_vdirs = get_vdirs_from_vdirsyncer_config()
+    if synced_vdirs:
+        print(f"Found {len(synced_vdirs)} calendars from vdirsyncer")
```

Comment:
Only print `"The following collections were found:"` is calendars are actually found:

```suggestion
    synced_vdirs = get_vdirs_from_vdirsyncer_config()
    if synced_vdirs:
        print("The following collections were found:")
        print(f"Found {len(synced_vdirs)} calendars from vdirsyncer")
```

There's mixed usage of "calendar" and "collection". Please stick to just one term and use that consistently.

Reply:
also fixed